backend/routes.js

```diff
@@ -229,7 +229,7 @@ export default class Servers {
             res.json(200, {http: HTTP_URL, https: HTTPS_URL});
         });
 
-        server.post(/\/oauth2\/token\/?$/, function saveOauth(req, res, next) {
+        server.post(/\/token\/?$/, function saveOauth(req, res, next) {
```

Comment:
can you add this redirect URI to the plotly admin? we'll also need to add it to the fixtures for on-premise

Reply:
or rather, the  `/ouath2` endpoint

Reply:
somewhere in the front-end code calls `/oath2/token` right now, it looks like that hasn't been changed yet.

Reply:
@chriddyp : 

  - Updated the redirect_uris at plot.ly/admin
  - Changed `POST` endpoint from `./token` to `./oauth2`(both in backend and FE)
  - Created an issue to change redirect_uri in fixtures https://github.com/plotly/streambed/issues/10070